database/factories/OrderFactory.php

```diff
@@ -1,16 +1,15 @@
 <?php
 
 use IndieHD\Velkart\Contracts\OrderRepositoryContract;
-use Ramsey\Uuid\Uuid;
+use IndieHD\Velkart\Contracts\ShoppingCartRepositoryContract;
 
 $order = resolve(OrderRepositoryContract::class);
+$cart = resolve(ShoppingCartRepositoryContract::class);
```

Comment:
Do we really want our cart implementation be named `ShoppingCart` vs simply just `Cart` ? It will be in a different namespace so shouldn't be concerned with conflicts.

Reply:
That's a fair point. Shorter is better. I will change it.